proto/pdpb2.proto

```diff
@@ -0,0 +1,298 @@
+syntax = "proto3";
+package pdpb2;
+
+import "metapb.proto";
+import "eraftpb.proto";
+
+import "gogoproto/gogo.proto";
+
+option (gogoproto.sizer_all) = true;
+option (gogoproto.marshaler_all) = true;
+option (gogoproto.unmarshaler_all) = true;
+
+service PD {
+    rpc Tso(TsoRequest) returns (TsoResponse) {}
+
+    rpc Bootstrap(BootstrapRequest) returns (BootstrapResponse) {}
+
+    // TODO: Carry it by rpc Bootstrap? Since a BOOTSTRAPPED Error is already
+    // definded in ResponseHeader.
+    rpc IsBootstrapped(IsBootstrappedRequest) returns (IsBootstrappedRequest) {}
```

Comment:
Carry it by rpc Bootstrap? Since a BOOTSTRAPPED Error is already definded in ResponseHeader.